wyoming_whisper/README.md

```diff
@@ -0,0 +1,27 @@
+# Wyoming Whisper
```

Comment:
Our other add-ons have pretty much the same file/readme structure. Maybe apply that one here too?

Additionally, DOCS.md is missing, which is shown in the add-on panel